dependency-check-suppressions.xml

```diff
@@ -103,6 +103,21 @@
         <cve>CVE-2019-7722</cve>
     </suppress>
 
+    <!-- The feign clients are not exposed so not an issue. -->
```

Comment:
Is this comment correct? Aren't the CVEs caused by AppInsights dependencies not Feign dependencies?